app/src/userpages/components/DataUnionPage/Item.jsx

```diff
@@ -378,11 +388,73 @@ const Item = ({ product, stats }: Props) => {
         setIsOpen(!isOpen)
     }, [isOpen])
 
+    const validate = useCallback((product) => {
+        const invalidFields = validateProduct(product, isEthIdentityRequired)
+        const errors = Object.keys(invalidFields)
+            .filter((field) => !!invalidFields[field])
+            .map((field) => field)
+
+        if (errors.length > 0) {
```

Comment:
Maybe better to show one notification in this view instead of all possible missing fields 🤔

Reply:
https://linear.app/streamr/issue/FRONT-421/show-only-one-notification-when-invalid-du-is-being-deployed-from-list